packages/legend-application-studio/style/components/editor/mapping-editor/_property-mapping-editor.scss

```diff
@@ -37,14 +37,30 @@
     margin-left: 0.5rem;
   }
 
-  &__name {
+  &__name--with-validation {
```

Comment:
this looks strange, this means you would have something like `__name--with-validation__name`, let's not do that, might as well call this `__name`, `__name__label` and only use the class `--with-validation` conditionally